### ISSUE TITLE
Can't put a small icon in front of menu item

### ISSUE THREAD

**Issue by TongZhai** [DotSpatial/DotSpatial#1346]
I haven't been successful in trying to put small icon in front of each dropdown menu item. The `SimpleActionItem` has `SmallImage` and `LargeImage`, I tried to set those, `SmallImage` replaces the menu text, and `LargeImage` doesn't do anything. Is this by design or is there another way?

**Comment by sindizzy** [comment]
try with another supplied image and see if that works.

**Comment by TongZhai** [comment]
@sindizzy do you mean it might work with a specific image format/data file? could you elaborate a bit more?

**Comment by jany-tenaj** [comment]
@TongZhai I don't think it will work. It looks as if Menus shouldn't have icons by design.

**Comment by TongZhai** [comment]
@jany-tenaj The DemoMap program actually has icons in front of menu items as shown below:

![image](https://user-images.githubusercontent.com/16242714/82784824-51b9bd00-9e2f-11ea-8571-8006dec8403e.png)

I was hoping the small and large image properties would be it,.. I guess I'll take a look into how that's done.

**Comment by jany-tenaj** [comment]
@TongZhai Checkout [MenuBarHeaderControl](https://github.com/DotSpatial/DotSpatial/blob/master/Source/DotSpatial.Controls/Header/MenuBarHeaderControl.cs#L119).

There you can see that you can add menu items with images to the ApplicationMenuKey.

**Comment by TongZhai** [comment]
@jany-tenaj I saw the line for `DisplayStyle`, I changed it to `ImageAndText`, but it puts the image next to the menu text, **NOT** in that left-hand column.
